src/lib/parser.c

```diff
@@ -303,6 +306,10 @@ __Z_INLINE parser_error_t parser_getDynamicItem(parser_context_t *ctx,
             break;
         case registryDeregisterEntity:
             return parser_ok;
+        case registryUnfreezeNode:
+            snprintf(outKey, outKeyLen, "Node ID");
```

Comment:
Not checking for `displayDynamicIdx == 0` here may result in any index to return nodeId